nailgun/entities.py

```diff
@@ -3253,6 +3253,62 @@ def __init__(self, server_config=None, **kwargs):
             'server_modes': ('sat'),
         }
 
+    def path(self, which=None):
+        """Extend ``nailgun.entity_mixins.Entity.path``.
+
+        The format of the returned path depends on the value of ``which``:
+
+        add_subscriptions
+            /hosts/<id>/add_subscriptions
+        remove_subscriptions
+            /hosts/<id>/remove_subscriptions
+
+        ``super`` is called otherwise.
+
+        """
+        if which in (
+                'add_subscriptions',
+                'remove_subscriptions'):
+            return '{0}/{1}'.format(
+                super(HostSubscription, self).path(which='base'),
+                which
+            )
+        return super(HostSubscription, self).path(which)
+
+    def add_subscriptions(self, synchronous=True, **kwargs):
```

Comment:
Would it make sense to have a parameter that accepts a subscription_entity & maybe quantity? That way it'd wouldn't require the user to check `apidoc` to work out what params are required.

Reply:
I believe that we had something like this before, but then we decided to move to one, singular signature for all methods across entire nailgun. That approach definitely has its own pros and cons.
Let's hear what other guys think about that

Reply:
If i remember correctly, main reasons we moved from that approach were:
- lots of commands have tens of accepted args, making method signature look awful. And sometimes method body too - usually you have to pass all those args
- (more importantly) fields in api change a lot, such approach will mean we should always keep all the args up-to-date. Updating entity's `_fields` is already a headache at times, and updating even more code per each api change will make it even worse

So the decision was to pass kwargs as it and make it user's responsibility to keep an eye on apidoc when writing tests.
We can think of possible options to improve user experience, but i'd like to take into consideration developer's experience as well :)

Reply:
I get your point and we can discuss that, but preferably in a separate issue or mailing thread and definitely not just two of us should make a decision, since it's pretty global change and we should let everyone vote.
In the meantime, let's not block this PR as this discussion is basically out of scope, it should be a separate effort to update args for all such helper methods if we decide to do so.

Reply:
I don't know any better place :( Maybe try mentioning people so they'll receive notification and will be more encouraged to participate?